modules/post/windows/manage/exec_powershell.rb

```diff
@@ -47,7 +47,7 @@ def run
     raise "Powershell not available" if ! have_powershell?
 
     # Preprocess the Powershell::Script object with substitions from Exploit::Powershell
-    script = make_subs(read_script(datstore['SCRIPT']),process_subs(datstore['SUBSTITUTIONS']))
+    script = make_subs(read_script(datstore['SCRIPT']),process_subs(datastore['SUBSTITUTIONS']))
```

Comment:
Fixed in https://github.com/rapid7/metasploit-framework/commit/0784370b9879ce0f2eeef496df11f2aa672fbaa3.